src/apps/loyalty/server/index.tsx

```diff
@@ -20,15 +20,19 @@ import { RelayMiddleware } from "./relay"
 
 const app = express()
 const artsyPassport = require("artsy-passport")
+const sharify = require("sharify")
```

Comment:
We would need to create typings for those, which is doable but not super necessary for now